presto-main/src/main/java/com/facebook/presto/geospatial/GeoFunctions.java

```diff
@@ -1341,4 +1343,46 @@ public Slice next()
             }
         };
     }
+
+    @SqlNullable
+    @Description("Decodes a GMaps-Polyline string into an array of Points")
+    @ScalarFunction("gmaps_polyline_decode")
+    @SqlType("array(" + GEOMETRY_TYPE_NAME + ")")
+    public static Block gMapsPolylineDecode(@SqlType(StandardTypes.VARCHAR) Slice polyline) throws PrestoException
```

Comment:
I believe this is implemented according to a spec or documentation.  If so, it would be helpful to link that documentation here.

Reply:
Yes. I have added a link to https://developers.google.com/maps/documentation/utilities/polylinealgorithm (this was in the Feature Request issue, but I agree that involves too much code archaeology to find via git blame -> git hub.